<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XXLProjectClass</class>
 <widget class="QMainWindow" name="XXLProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHelp">
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRank">
       <property name="text">
        <string>排行榜</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
